client_code/MultiSelectDropDown/__init__.py

```diff
@@ -228,6 +224,45 @@ def format_selected_text(self, count, total):
         return ", ".join(opt.title or opt.key for opt in self._options if opt.selected)
 
     ##### PROPERTIES #####
+    @property
+    def background(self):
+        return self._background
+
+    @background.setter
+    def background(self, value):
+        self._background = value
+        if value:
+            _S(_js.get_dom_node(self._select_btn)).children("button").css(
+                "background-color", value
+            )
+
+        pop(self._select_btn, "destroy")
+        popover(
+            self._select_btn,
+            self._dd,
+            placement="bottom-start",
+            arrow=False,
+            delay=0,
+            animation=False,
+            trigger="manual",
+            max_width="fit-content",
+            background=value,
+        )
+
+    @property
+    def foreground(self):
+        return self._foreground
+
+    @foreground.setter
+    def foreground(self, value):
+        self._foreground = value
+        if value and self._props_initialized:
```

Comment:
It doesn't seem like it's possible to remove background/foreground

Reply:
did that by just removing the conditional checks (for the most part)